packages/smooth_app/lib/helpers/collections_helper.dart

```diff
@@ -132,4 +132,7 @@ extension MapStringKeyExtensions<V> on Map<String, V> {
     final String? mapKey = keyStartingWith(key, ignoreCase: ignoreCase);
     return this[mapKey];
   }
+
+  Map<String, V> where(bool Function(MapEntry<String, V>) fn) =>
+      Map<String, V>.fromEntries(entries.where(fn));
```

Comment:
```suggestion
  Map<String, V> where(bool Function(MapEntry<String, V>) check) =>
      Map<String, V>.fromEntries(entries.where(check));
```

Beyond the political connotation of `fn`, it looks like we're either obfuscating our own code, or coding in another language, or coding in another century.